react/more_react_concepts/refs_and_memoization.md

```diff
@@ -319,4 +319,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- The `memo` function is straightforward that's why we didn't cover it too much, but if you want to know more about it, especially if you want to create your own logic for the `memo` function go to the [documentation for memo](https://react.dev/reference/react/memo).
+- The `memo` function is straightforward that's why we didn't cover it too much, but if you want to know more about it, especially if you want to create your own logic for the `memo` go to the documentation for [memo wrapper function](https://react.dev/reference/react/memo).
```

Comment:
Aha, brilliant spot regarding the "same link text for same href" thing! :rocket: 
This definitely does not need duplication in the AR section, so could you replace it with the following please?
```suggestion
- - It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```